redfish-core/lib/fabric_adapters.hpp

```diff
@@ -236,6 +238,45 @@ inline void doAdapterGet(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
     }
 }
 
+inline void doAdapterPatch(const crow::Request& req,
+			   const std::shared_ptr<bmcweb::AsyncResp>& aResp,
+			   const std::string& adapterId,
+			   const std::string& fabricAdapterPath)
+{
+    std::optional<bool> locationIndicatorActive;
+        
+    if (!json_util::readJsonPatch(req, aResp->res,
+			     "LocationIndicatorActive",
+			     locationIndicatorActive))
+    {
+      return;
+    }
+    
+    crow::connections::systemBus->async_method_call(
+       [fabricAdapterPath, adapterId,locationIndicatorActive, aResp](
+	 const boost::system::error_code ec) {
+	 if (ec)
+	 {
+	     handleAdapterError(ec, aResp->res, adapterId);
+	     return;
+	 }
+	 
+	 if (locationIndicatorActive)
+	 {
+	     setLocationIndicatorActive(
+					aResp, fabricAdapterPath,
+					*locationIndicatorActive);
+	 }
+    
+	},
+	"xyz.openbmc_project.ObjectMapper",
+	"/xyz/openbmc_project/object_mapper",
+	"xyz.openbmc_project.ObjectMapper", "GetSubTree",
```

Comment:
doAdapterPatch already did a GetSubTree? Don't you already know the fabricAdapterPath ?

Reply:
This is resolved in the change to the issues below

Reply:
This has been removed in latest commit.